.github/workflows/build_windows.yml

```
@@ -349,45 +351,49 @@ jobs:
-      uses: jurplel/install-qt-action@v4
-      with:
-        version: 6.10.2
-        host: 'windows'
-        target: 'desktop'
-        arch: 'win64_msvc2022_64'
-        modules: 'qt5compat qtnetworkauth qtshadertools qtwebsockets'
-        cache: true
-    - name: Setup environment
-      run: |
-        buildscripts\ci\windows\setup.bat --portable ON
+    - parallel:
```

actionlint (1.7.12)

[error] step must run script with "run" section or run action with "uses" section

(syntax-check)